README.md

```diff
@@ -20,11 +20,11 @@ Then add a configuration file to your project root. Here is an example:
 
 ### screener.config.js
 
+#### Using API Key to authenticate
```

Comment:
Suggest adding "Screener" to be a bit more explicit: `Using Screener's API Key to authenticate`

Reply:
Done